TripPlanner/View Controllers/LoginViewController.m

```diff
@@ -22,6 +22,29 @@ - (void)viewDidLoad {
     // Do any additional setup after loading the view.
 }
 
+// enable log in on button click
+- (IBAction)onSignIn:(id)sender {
+    NSString *username = self.usernameField.text;
+    NSString *password = self.passwordField.text;
+    
+    [PFUser logInWithUsernameInBackground:username password:password block:^(PFUser * user, NSError *  error) {
+        
+        if (error != nil) {
+            UIAlertController *loginAlert = [self createAlert:error.localizedDescription title:@"Error Logging In"];
+            
+            // show the alert controller
+            [self presentViewController: loginAlert animated:YES completion:^{
+                // optional code for what happens after the alert controller has finished presenting
+            }];
+        } else {
+            NSLog(@"User logged in successfully");
```

Comment:
nit: putting the success condition first makes the code a bit easier to read (i.e. swapping the if-else conditions here)